components/teletask.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "teletask",
+    "owner": [
+        "@tiemooowh"
+    ],
+    "manifest": "https://github.com/Tiemooowh/homeassistant-teletask/blob/main/custom_components/teletask/manifest.json",
```

Comment:
Use the raw URL, this links to a GitHub page, not the `json` file.

Reply:
corrected to the raw url, sincere appologies